core/config/locales/templates/dashboard/provisioning/en.yml

```diff
@@ -70,11 +70,11 @@ en:
             status: "Status"
             states:
               requested: requested
-              approved: approved
```

Comment:
No longer a need for approved?

Reply:
The `approved` state has been removed, the transition is now from called `approve` and transition from `requested` to `provisioning`